config/routes.rb

```diff
@@ -27,6 +27,11 @@
     end
   end
 
-  resources :issues, only: [:new, :create, :show]
+  resources :issues, only: [:new, :create, :show] do
+    resources :comments, module: :issues, only: [:create, :destroy]
```

Comment:
OK.
According those settings you will have url with :issue_id and :id parameters.
Why you need :issues_id in url if you already have :id for comment. :id is enough to find comment and destroy it.
Also, as I see, below you have another destroy url.
resources :comments, only: [:destroy]
Looks like you have two urls for the same thing. Do you have any reason for that?
But anyway, pleas take a look at [shallow](http://edgeguides.rubyonrails.org/routing.html#shallow-nesting)